tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -316,27 +321,49 @@ def do_test(self, test):
         return (test.result_cls(*data), [])
 
     def do_testharness(self, protocol, url, timeout):
-        format_map = {"abs_url": url,
-                      "url": strip_server(url),
-                      "window_id": self.window_id,
-                      "timeout_multiplier": self.timeout_multiplier,
-                      "timeout": timeout * 1000}
+        format_map = {"url": strip_server(url)}
 
         parent_window = protocol.testharness.close_old_windows()
         # Now start the test harness
-        protocol.base.execute_script(self.script % format_map, async=True)
-        test_window = protocol.testharness.get_test_window(self.window_id, parent_window)
-
+        protocol.base.execute_script("window.open('about:blank', '%s', 'noopener')" % self.window_id)
```

Comment:
Note that Edge will shortly be using this and doesn't support `noopener`.

Reply:
I"m not super worried about this; any test that depends on the opener being null at the moment is already broken. I only used noopener to prove that we aren't depending on the opener any more,

In the medium term I hope that everyone will implement the `Create Window` webdriver endpoint, which is inteded to have exactly the behaviour we require here and also returns the handle of the new window so we can delete the window finding code. I'm happy to do the more complex solution in the interim if people think that it's important, but otherwise I would tend to argue that this good enough for now.

Reply:
I don't think we need to worry about older versions of Safari; it's almost exclusively Edge that's my concern here.

Reply:
let's try explicitly setting window.open().opener = null, because I don't think that can make anything worse?

Reply:
@jgraham https://gist.github.com/fb3941b207c8d9ff3f3343a7bf302165 (this is with Selenium)

Reply:
@jgraham https://gist.github.com/c939fd3a4276f3bc056e175e483e57a5 (edge_webdriver)